MapData.py

```diff
@@ -17,7 +16,6 @@ def __init__(self):
         self.__setattr__(RADIUS, 0.1)
         self.__setattr__(IMAGE, None)
         self.__setattr__(MARK, None)
-        self.__setattr__(LOCATION, None)
```

Comment:
Becoming a local var in MapThread then? Was that part of the cause?

Reply:
I actually removed it because there aren't any usages. I can't remember where it was used before, but we much have changed something in the meantime and now its just not used.